package.json

```diff
@@ -1,33 +1,48 @@
 {
-  "name": "team1-frontend",
-  "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
-  "scripts": {
-    "start": "npx tsc && cp -R views dist && node dist/app.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "author": "",
-  "license": "ISC",
-  "dependencies": {
-    "@types/express-session": "^1.17.7",
-    "axios": "^1.5.0",
-    "express": "^4.18.2",
-    "express-session": "^1.17.3",
-    "node": "^20.6.0",
-    "nunjucks": "^3.2.4"
-  },
-  "devDependencies": {
-    "@types/express": "^4.17.17",
-    "@types/node": "^20.5.7",
-    "typescript": "^5.2.2"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/CAP-M-McD/Team1_Frontend.git"
-  },
-  "bugs": {
-    "url": "https://github.com/CAP-M-McD/Team1_Frontend.git/issues"
-  },
-  "homepage": "https://github.com/CAP-M-McD/Team1_Frontend.git#readme"
+    "name": "team1-frontend",
+    "version": "1.0.0",
+    "description": "",
+    "main": "app.js",
+    "scripts": {
+        "start": "npx tsc && cp -R views dist && cp -R public dist && node dist/app.js",
+        "test": "echo \"Error: no test specified\" && exit 1",
+        "lint": "htmlhint *.html **/*.html && stylelint *.css **/*.css && eslint *.ts **/*.ts && prettier . --check",
```

Comment:
Could you add a section to the `README.md` file to outline how to run the lint check and how to fix the linter issues.

Could you also add the lint check to run before running a commit as a git hook?

Reply:
updated readme and added pre-commit hook